test/SampleService_test.py

```diff
@@ -2006,6 +2005,83 @@ def _update_acls(url, token, params, print_resp=False):
     assert ret.json() == {'version': '1.1', 'id': '67', 'result': None}
 
 
+def _update_samples_acls(url, token, params, print_resp=False):
```

Comment:
I originally did all the service calls by hand since the sample service was a dynamic service and therefore the client was configured to try and look up the service url with the service wizard, which obviously didn't work for integration tests. 

Since the sample service is no longer a dynamic service we should probably remove the dynamic service flag from `kbase.yml`, recompile the client, and use that instead of rolling the calls ourselves.

out of scope for this PR though.